all-sdk/external-client-configuration.md

```diff
@@ -337,7 +344,7 @@ is SDK specific).
         * This is the new way and we should not try to support both at the same time, it can get confusing.
   * The default is `default`, and this can also be set via `TEMPORAL_PROFILE` env var.
 * CLI will accept `--config-file`.
-  * Default is `~/.config/temporalio/temporal-client.toml`, also can be set via `TEMPORAL_CONFIG_FILE` env var.
+  * Default is `~/.config/temporalio/temporal.toml`, also can be set via `TEMPORAL_CONFIG_FILE` env var.
```

Comment:
This is kinda specified above at:

> The default config file location is `<app-config-dir>/temporalio/temporal.toml` for all platforms. [...] This is as defined by https://pkg.go.dev/os#UserConfigDir which, as of this writing, is the logic defined in docs and in code at https://cs.opensource.google/go/go/+/refs/tags/go1.23.0:src/os/file.go;l=528.

And there's a hint before in one of the flows as sample instructions to users:

* macOS - `$HOME/Library/Application Support/temporalio/temporal.toml`
* Windows - `%AppData%/temporalio/temporal.toml` (often `C:\Users\<myuser>\AppData\Roaming\temporalio\temporal.toml`)
* Linux - `$HOME/.config/temporalio/temporal.toml`

But I can take that part and make it clearer here if needed.

Reply:
Yeah, it's clearly indicated _elsewhere_, and I would normally not be picky on this kind of small incoherencies.

But this specific line sounds like it is authoritative, so being incomplete in this case is more confusing.

TBH, I'd just remove this line, or formulate it as an example rather than a prescriptive statement.

Reply:
Removed the part saying what the default is here under the CLI section hoping people will look at the general specification section